CHANGELOG.md

```diff
@@ -15,6 +15,18 @@ For reference, the possible headings are:
 * **External Contributors** to list contributors outside of BigchainDB GmbH.
 * **Notes**
 
+
+## [1.0.1] - 2017-07-12
+Tag name: v1.0.1
+
+### Fixed
+* Mistakenly changed MongoDB version via PR [#1641](https://github.com/bigchaindb/bigchaindb/pull/1641).
+* Quickstart guide via PR [#1648](https://github.com/bigchaindb/bigchaindb/pull/1648).
+* Issue [#1627](https://github.com/bigchaindb/bigchaindb/issues/1627).
+* Issue [#1645](https://github.com/bigchaindb/bigchaindb/issues/1645).
+* Issue [#1657](https://github.com/bigchaindb/bigchaindb/issues/1657).
```

Comment:
I usually list the pull requests rather than the issues, but this is probably okay, since GitHub links the issues to the PRs which resolved them automatically. FYI, 1638 resolved 1627, 1646 resolved 1645, and 1659 resolved 1657.